api/test/e2enew/ssl/ssl_suite_test.go

```diff
@@ -25,12 +25,11 @@ import (
 	"e2enew/base"
 )
 
-func TestRoute(t *testing.T) {
+func TestSSL(t *testing.T) {
 	ginkgo.RunSpecs(t, "ssl suite")
 }
 
 var _ = ginkgo.AfterSuite(func() {
 	base.CleanResource("ssl")
-	base.CleanResource("routes")
```

Comment:
@starsz @nic-chen 
I have a question here. If we create a route in the test case and delete the route, and verify that the route is indeed deleted, do we still need to executeAfterSuite(base.CleanResource("resource") to ensure the deletion of resources?

Reply:
I think we don't need to do `execute AfterSuite(base.CleanResource("resource") `

Reply:
By the way, I remove the response check. cc @nic-chen 

https://github.com/apache/apisix-dashboard/pull/1571/files#diff-a15775a07088498daed7a85df9b271a05806905c320e38029391c37bef35fb37L277

Reply:
ok

Reply:
OK